src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -124,6 +129,18 @@ public ResponseEntity<String> updateMeasure(
     return response;
   }
 
+  @GetMapping("/measures/{measureId}/groups")
+  public ResponseEntity<List<Group>> getGroups(@PathVariable String measureId) {
```

Comment:
is this something unrelated?

Reply:
yeah this is unrelated to the story - I added it because I was in the controller, Brandon had been asking about an endpoint to fetch measure groups for testing purposes, and it was quick to add one. And as a subresource that we have endpoints exposed for creating and updating, having an endpoint to fetch the subresources also is not out of line with REST standards.

Reply:
got it. Thanks for the clarification!